content/en/agent/proxy.md

```diff
@@ -504,6 +504,15 @@ backend datadog-network-devices-netflow
     server-template mothership 5 ndmflow-intake.{{< region-param key="dd_site" >}}:443 check port 443 ssl verify required ca-file <PATH_TO_CERTIFICATES> check resolvers my-dns init-addr none resolve-prefer ipv4
     # Uncomment the following configuration for older HAProxy versions
     # server mothership ndmflow-intake.{{< region-param key="dd_site" >}}:443 check port 443 ssl verify required ca-file <PATH_TO_CERTIFICATES>
+
+#remove this section if your environment does not support Remote Configuration
+backend datadog-remote-configuration
```

Comment:
I think we may need a `frontend` section to match these HAProxy sections here and in HTTPS section

Reply:
@arbll can you please confirm if we need a frontend section also per Srdjan's comment above?